src/hmrc-content-guide/anti-dumping-measures/index.njk

```diff
@@ -57,6 +72,16 @@ layout: twoColumnPage.njk
 
 <p class="govuk-body">Enter the contents of the Anti-Dumping Duty measure.</p>
 
+<h3 class="govuk-heading-m">Tagging and collection page</h3>
+
+<p class="govuk-body">Make sure you:</p>
+<ul class="govuk-list govuk-list--bullet">
+<li>add the appropriate associations and specialist sector tags</li>
+<li>add the new measure to the <a href="https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.gov.uk%2Fgovernment%2Fcollections%2Fanti-dumping-duty-measures&data=04%7C01%7Cdaniela.salvini%40hmrc.gov.uk%7C8caf0b75baf746fc940308d88a364a2a%7Cac52f73cfd1a4a9a8e7a4a248f3139e1%7C0%7C0%7C637411313057702957%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=XwxvROvDxGpSBfcF97f%2FjnBb5AZQO1AwndLftZwVRyA%3D&reserved=0" class="govuk-link">anti-dumping duty measures</a> document collection - do not create a public change note</li>
```

Comment:
The href looks different to the one in the previous PR. Is this the correct one?